bsdlib/doc/security_tags.rst

```diff
@@ -0,0 +1,51 @@
+.. _security_tags:
+
+Security tags
+#############
+
+The modem on an nRF9160 device is equipped with a full IPv4/IPv6 stack and a DTLS/TLS engine.
+To use the cryptographic functions in the modem, the application must provision the security credentials to the modem.
+
+To be able to provision credentials, the modem must be offline or in offline mode.
```

Comment:
`offline or in offline` looks incorrect.

Reply:
I don't know what the original intention was, but I interpreted it in a way that "offline" and "offline mode" are two different things - my guess was that the modem is "offline" when it didn't manage to establish a connection, and that it is in "offline mode" when you explicitly tell it to go offline with AT commands.
Does that make any sense, or should we just change it to "must be in offline mode"?